main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def get_race(race_info: dict) -> Race:
+    result = Race.objects.get_or_create(
+        name=race_info["name"],
+        description=race_info["description"]
+    )[0]
```

Comment:
result is bad name for variable, also it is better to use underscore here, instead of indexes ))

Reply:
same is for similar cases